src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.stories.js

```diff
@@ -0,0 +1,68 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+
+const iconName = {
+  control: {
+    type: 'text',
+  },
+};
+
+const id = {
+  control: {
+    type: 'text',
+  },
+};
+
+const defaultArgTypes = {
+  'icon-name': iconName,
+  id,
+};
+
+const defaultArgs = {
+  'icon-name': 'circle-information-small',
+  id: 'tooltip-trigger',
+};
+
+const tooltip = () => (
+  <sbb-tooltip data-testid="tooltip" trigger="tooltip-trigger">
+    <p id="tooltip-content" style={'margin: 0; font-size: var(--sbb-font-size-text-s);'}>
+      Simple information tooltip with link.
+    </p>
+  </sbb-tooltip>
+);
+
+const Template = (args) => [<sbb-tooltip-trigger {...args}></sbb-tooltip-trigger>, tooltip()];
```

Comment:
Please wrap Text around tooltip trigger and make sure that it aligns correctly with the text.

NOK: 
<img width="190" alt="image" src="https://user-images.githubusercontent.com/29278232/211475927-6758c629-5cb0-4e2d-981e-7110a22afe73.png">